tools/cpp/MNNV2Basic.cpp

```diff
@@ -184,15 +190,10 @@ static int test_main(int argc, const char* argv[]) {
     }
     MNN_PRINT("\n");
 
-    int precision = BackendConfig::Precision_Low;
-    if (argc > 7) {
-        precision = atoi(argv[7]);
-    }
-
     // create net
     MNN_PRINT("Open Model %s\n", fileName);
     std::shared_ptr<MNN::Interpreter> net =
-        std::shared_ptr<MNN::Interpreter>(MNN::Interpreter::createFromFile(fileName));
+        std::shared_ptr<MNN::Interpreter>(MNN::Interpreter::createFromFile(fileName), MNN::Interpreter::destroy);
```

Comment:
This code is opening the MNN model file.
How can we intergrate to ipfs-lite-cpp proj?